Content.Server/DeltaV/StationEvents/Events/DebrisSpawnerRule.cs

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
This is completely fucking shitcode, literally copied from the original Pirate Radio System that predates when I learned Trigonometry. By god this abomination should be deleted in its entirety.